packages/create-keystone-app/example-projects/config.js

```diff
@@ -1,14 +1,20 @@
+const slugify = require('@sindresorhus/slugify');
+
 const adapters = {
-  Mongoose: {
+  MongoDB: {
+    name: 'MongoDB',
     file: 'adapter-mongoose.js',
     dependencies: ['@keystonejs/adapter-mongoose'],
-    description: 'Connect to a Mongo database.',
+    description: 'Connect to a MongoDB database.',
+    defaultConfig: name => `mongodb://localhost/${slugify(name)}`,
```

Comment:
This has a different slugify method to the postgres example, is that consistent with how we slugify the project names for default connections strings?

I'm going to assume it is and that the reason is mongo names can have `-` but postgres can't?

Maybe not for this PR but we should totally just make that consistent and always `_`. Unfortunately that's breaking :(

Reply:
Your have surmised correctly 👍